src/main/java/org/broadinstitute/hellbender/tools/spark/sv/BreakpointAllele.java

```diff
@@ -0,0 +1,62 @@
+package org.broadinstitute.hellbender.tools.spark.sv;
+
+import org.broadinstitute.hellbender.utils.SimpleInterval;
+
+import java.util.List;
+import java.util.Objects;
+
+/**
+ * This class represents the allele of an SV breakpoint (a novel adjacency between two genomic locations
+ */
+class BreakpointAllele {
+    final SimpleInterval leftAlignedLeftBreakpoint;
+    final SimpleInterval leftAlignedRightBreakpoint;
+    final String insertedSequence;
+    final String homology;
+    final boolean fiveToThree;
+    final boolean threeToFive;
```

Comment:
What about replacing these two `boolean`'s with a 
`final InversionType inversionType;`, and initialize it with 
```java
if(!fiveToThree && threeToFive){
    return InversionType.INV_3_TO_5;
} else if(fiveToThree && !threeToFive) {
    return InversionType.INV_5_TO_3;
} else {
    return InversionType.INV_NONE;
}
```